hw6/hw6/ViewControllers/AcademyAwardsViewController.swift

```diff
@@ -0,0 +1,151 @@
+        view.addSubview(tableView)
+        setTableViewDelegates()
+        
+        tableView.rowHeight = 100
+        tableView.pin(to: view)
+        
+        tableView.register(CustomCell.self, forCellReuseIdentifier: CellIdentifier.actors.rawValue)
+        tableView.register(CustomCell.self, forCellReuseIdentifier: CellIdentifier.films.rawValue)
+    }
+    
+    func setTableViewDelegates() {
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    func tableView(
+        _ tableView: UITableView,
+        numberOfRowsInSection section: Int
+    ) -> Int {
+        if section == 0 {
+            return actors.count
+        } else {
+            return films.count
+        }
+        
+    }
+    
+    
+    enum CellIdentifier: String {
+        case actors
+        case films
+    }
+ 
+    
+    func tableView(
+        _ tableView: UITableView,
+        cellForRowAt indexPath: IndexPath
+    ) -> UITableViewCell {
+        if indexPath.section == 0 {
+            
+            let cell: UITableViewCell
+            
+            if let dequeueReusableCell = tableView.dequeueReusableCell(withIdentifier: CellIdentifier.actors.rawValue) {
+                cell = dequeueReusableCell
+                
+            } else {
+                cell = UITableViewCell(style: .default, reuseIdentifier: CellIdentifier.actors.rawValue)
+            }
+            let actors = actors[indexPath.row]
+            
+            var configuration = UIListContentConfiguration.valueCell()
+            configuration.text = actors.actor
+            
+            if indexPath.row == 1{
+                configuration.secondaryText = "Joker"
+            }
+            if indexPath.row == 3{
+                configuration.secondaryText = "Forest Gump"
+            }
+            
+            configuration.image = actors.image
+            cell.contentConfiguration = configuration
+            
+            return cell
+            
+        } else {
+            
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: CellIdentifier.films.rawValue, for: indexPath) as? CustomCell else{
+                fatalError("Could not deque cell of type \(CustomCell.self)")
+            }
+            
+            let film = films[indexPath.row]
+            cell.set(films: film)
+            return cell
+        }
+    }
+}
+
+extension AcademyAwardsViewController {
+    func fetchData() -> [Actors] {
+        let actor1 = Actors(image: Images.forestWhitaker, actor: "Forest Whitaker")
+        let actor2 = Actors(image: Images.joaquinPhoenix, actor: "Joaquin Phoenix")
+        let actor3 = Actors(image: Images.matthewMcConaughey, actor: "Matthew McConaughey")
+        let actor4 = Actors(image: Images.tomHanks, actor: "Tom Hanks")
+        let actor5 = Actors(image: Images.leonardoDiCaprio, actor: "Leonardo DiCaprio")
+        
+        return [actor1, actor2, actor3, actor4, actor5]
+    }
```

Comment:
Можно инит вызывать прямо в Array.

```suggestion
    func fetchData() -> [Actors] {
        [
            Actors(image: Images.forestWhitaker, actor: "Forest Whitaker"),
            Actors(image: Images.joaquinPhoenix, actor: "Joaquin Phoenix"),
            Actors(image: Images.matthewMcConaughey, actor: "Matthew McConaughey"),
            Actors(image: Images.tomHanks, actor: "Tom Hanks"),
            Actors(image: Images.leonardoDiCaprio, actor: "Leonardo DiCaprio"),
        ]
    }
```